src/app/components/generics/dialogs/latex-editor-dialog.component.html

```diff
@@ -1,48 +1,50 @@
 <mat-dialog-content class="dialog-container">
   <h1 mat-dialog-title id="title">{{ data.title }}</h1>
-  <mat-card class="card-container">
+  <mat-card appearance="outlined" class="card-container">
     <div>Default LaTeX packages:</div>
     <div *ngFor="let defaultPackage of defaultLatexPackages">
       <mat-label style="font-weight: bold;">{{ defaultPackage }}</mat-label>
     </div>
   </mat-card>
-  <mat-card class="card-container">
+  <mat-card appearance="outlined" class="card-container">
     <mat-form-field class="content-container">
       <mat-label class="input-label">Additional LaTeX packages</mat-label>
       <textarea
         class="content-container"
         [(ngModel)]="latexPackages"
-        [matAutosizeMaxRows]="3"
-        [matAutosizeMinRows]="2"
-        [matTextareaAutosize]="true"
+        cdkAutosizeMaxRows="3"
```

Comment:
a maximum of 3 rows seems a little too small for larger latex content this is intended for. Also the dialog renders way too thin...